src/test/java/teammates/test/cases/action/InstructorFeedbackResponseCommentDeleteActionTest.java

```diff
@@ -28,6 +28,7 @@ protected String getActionUri() {
     @Override
     @Test
     public void testExecuteAndPostProcess() {
+        prepareTestData();
```

Comment:
What if you modify `BaseActionTest` instead to run `prepareTestData` before every test method?
```patch
-    @BeforeClass
+    @BeforeMethod
     public void baseClassSetup() {
         prepareTestData();
     }
```